lib/screens/dashboard/relativeScreen/relative.dart

```diff
@@ -0,0 +1,290 @@
+import 'package:flutter/material.dart';
+import 'package:staff_management/const_value/controller.dart';
+import 'package:staff_management/models/employee.dart';
+import 'package:staff_management/models/relative.dart';
+import 'package:staff_management/utils/dropdown/dropdownButton.dart';
+import 'package:syncfusion_flutter_datagrid/datagrid.dart';
+import 'package:intl/intl.dart';
+
+class RelativeScreen extends StatefulWidget {
+  const RelativeScreen({Key? key}) : super(key: key);
+
+  @override
+  _RelativeScreenState createState() => _RelativeScreenState();
+}
+
+class _RelativeScreenState extends State<RelativeScreen> {
+  late EmployeeDataSource employeeDataSource;
+  RelativeQuery _relativeQuery = RelativeQuery(year: "none");
+  String _selectedYear = "none";
```

Comment:
Should change default value to somethings else like "All relatives" => show all the relatives, maybe. "none" perhaps make user misread

Reply:
"All relatives" means also show those relatives whose type is "Vợ/Chồng"